src/pages/TradeV3/perps/components/CollateralPanel.tsx

```diff
@@ -206,177 +101,59 @@ const Accounts: FC<{ isSolAccount: boolean }> = ({ isSolAccount }) => {
-          <Tooltip color={mode === 'dark' ? '#F7F0FD' : '#1C1C1C'}>
-            The health bar shows how close you are to being liquidated.{' '}
-          </Tooltip>
-        </div>
-        {getHealthData()}
-      </ACCOUNT_ROW>
-      {/*<ACCOUNT_ROW $height={height}>
-        <div className="tooltip-row">
-          <span className="key">{isSolAccount ? 'Balance' : 'Balances'}</span>
-          <Tooltip color={mode === 'dark' ? '#EEEEEE' : '#1C1C1C'}>
-            The health bar shows how close you are to being liquidated.{' '}
-          </Tooltip>
-        </div>
-        {isSolAccount ? (
-          <span className="value">{currentMarketBalance}</span>
-        ) : (
-          <div className="balances value">
-            <div>${(Number(traderInfo.collateralAvailable) * Number(collateralInfo.price)).toFixed(2)}</div>
-            <div>0.3344 BTC</div>
-            <span className="value">1,888.55 USDC</span>
-          </div>
-        )}
-      </ACCOUNT_ROW>*/}
-      <ACCOUNT_ROW $height={height}>
-        <div className="tooltip-row">
-          <span
-            className={
-              (traderInfo && traderInfo.pnl && Number(traderInfo.pnl)
-                ? traderInfo.pnl[0] === '-'
-                  ? 'redText '
-                  : 'greenText '
-                : ' ') + 'key '
-            }
-          >
-            Unrealized P&L
-          </span>
-          <Tooltip color={mode === 'dark' ? '#F7F0FD' : '#1C1C1C'}>
-            The total profit and loss from your positions in your account{' '}
-          </Tooltip>
-        </div>
-        <span className="value">{pnl}</span>
-      </ACCOUNT_ROW>
-      <ACCOUNT_ROW $height={height}>
-        <div className="tooltip-row">
-          <span className="key">User Volume</span>
-          <Tooltip color={mode === 'dark' ? '#F7F0FD' : '#1C1C1C'}>
-            User volume refers to the total volume of trades made by you. The higher your volume, the greater your
-            chances of receiving rewards in the competition{' '}
-          </Tooltip>
-        </div>
-        <span className="value">{formatNumberInThousands(Number(userVolume))} $</span>
-      </ACCOUNT_ROW>
-      <ACCOUNT_ROW $height={height}>
-        <div className="tooltip-row">
-          <span className="key">Balance</span>
-          <Tooltip color={mode === 'dark' ? '#F7F0FD' : '#1C1C1C'}>
-            Balance refers to the total value of your cash balance that you can use as collateral for opening new
-            positions or maintaining existing ones. A negative balance indicates a notional position size greater
-            than the amount deposited.{' '}
-          </Tooltip>
+  const getBarsData = () => {
+    const accountHealth = 40
+    return (
+      <div className="h-[20px] w-[32px]">
+        <div className="flex">
+          {[0, 25, 50, 75].map((item, index) => (
+            <div
+              key={index}
+              className={
```

Comment:
this seems like a good time to use the cn, to compile this className in a more readable manner